staging/src/k8s.io/apimachinery/pkg/util/wait/wait_test.go

```diff
@@ -875,7 +875,7 @@ func TestExponentialBackoffWithContext(t *testing.T) {
 			}
 
 			attempts := 0
-			err := ExponentialBackoffWithContext(test.ctxGetter(), backoff, func() (bool, error) {
+			err := ExponentialBackoffWithContext(test.ctxGetter(), backoff, func(_ context.Context) (bool, error) {
```

Comment:
Interesting, I think we don't have any test that exercise the cancelation of a conditionfunc that blocks the wait loop , do you want to add one as follow up or here?